packages/shim-deno/package.json

```diff
@@ -38,6 +38,7 @@
     "update-progress": "deno run --allow-read tools/missing.ts > PROGRESS.md"
   },
   "dependencies": {
+    "@deno/kv": "^0.7.0",
```

Comment:
I don't think we should add this dependency here and instead we should document how someone can build up a `Deno` object that they can then import. For example, I believe the following would work:

```ts
// deno.ts
export * from "@deno/shim-deno";
export * from "@deno/kv";
```

Then do:

```ts
import * as Deno from "./deno.ts";
```

Reply:
🙈 so embarrassing, you're right, thanks for this.